packages/cna-template/template/frameworks/github-actions/.github/workflows/ci.yml

```diff
@@ -0,0 +1,66 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
+
+jobs:
+  ci:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node: [14]
+
+    steps:
+      - name: Checkout 🛎
+        uses: actions/checkout@master
+
+      - name: Setup node env 🏗
+        uses: actions/setup-node@v2.1.1
+        with:
+          node-version: ${{ matrix.node }}
+
+      <%_ if (pm === 'yarn') { _%>
+      - name: Get yarn cache directory path 🛠
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache node_modules 📦
+        uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      <%_ } _%>
+      <%_ if (pm === 'npm') { _%>
+      - name: Cache node_modules 📦
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      <%_ } _%>
+
+      - name: Install dependencies 👨🏻‍💻
+        run: <%= pm === 'yarn' ? 'yarn' : 'npm' %> ci
```

Comment:
Can be `pmRun`.

Reply:
- [x] done in c90f1e4

Reply:
Sorry that I may not explain clearly, pmRun is a wapper for both yarn and npm, the value is  `yarn` or `npm run`, so what we could do is:

```js
// From
<%= pm === 'yarn' ? 'yarn' : 'npm' %> ci
// To
<%= pmRun %> ci
```

Reply:
yes, sorry, I am half asleep due to sleep deprivation.. I've fixed it in e62774a